torch/_functorch/config.py

```diff
@@ -193,6 +193,10 @@
 # Used for tests
 strict_autograd_cache = False
 
+# Fx pass used to tensorify python scalars. Part of the rollout for supporting
+# dynamic float arguments in PT2.
+tensorify_python_scalars = True
```

Comment:
err, were you going to killswitch this?

Reply:
Sorry I think I misunderstood how JK's work within pytorch. I thought they would override these config flags by name. I put up https://github.com/pytorch/pytorch/pull/137490/files and will double check the integration w/ Colin before folding it back into this diff.